Tails specific: ttdnsd is started via NetworkManager hooks.
No need to properly stop it on shutdown.

--- chroot.orig/etc/init.d/ttdnsd	2012-09-24 10:05:13.333056580 +0200
+++ chroot/etc/init.d/ttdnsd	2012-09-24 10:03:30.638108333 +0200
@@ -6,2 +6,2 @@
-# Default-Start:     2 3 4 5
+# Default-Start:
-# Default-Stop:      0 1 6
+# Default-Stop:
